content/docs/querying/api.md

```diff
@@ -71,6 +71,7 @@ URL query parameters:
 
 - `query=<string>`: Prometheus expression query string.
 - `time=<rfc3339 | unix_timestamp>`: Evaluation timestamp. Optional.
+- `timeout=<duration>`: Evaluation timeout. Optional. Defaults to value of `-query.timeout` flag.
```

Comment:
Ooohhhh... good point. Hmm. Is that actually the behavior we want? I can imagine most users wanting to set a *higher* than default timeout for certain queries that take a long time, but which they still really want to get an answer to.